thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -611,6 +615,7 @@ def benchmark_main(return_metrics_as_json=False, json_path="", **kwargs) -> None
         # Helps catch OutOfMemory Errors and post processing of errors
         if global_rank in [0, None]:
             print("An error occurred:", type(error).__name__, "–", error)
+        raise
```

Comment:
if we do raise, what would be the benefit(s) of using `try-except`?

Reply:
I guess I tried to be compatible with previous version if Parth Mannan would like to parse the messages in the same way as previously, but actually it won't be compatible anyway, because the error will be raised and json results won't be saved. 
I've removed the try except block.